run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+for i in `ls $(readlink -m $0/../tests)`
+do
+	echo -n Running test $i...
+	if ! ./test --test $(readlink -m $0/../tests)/$i
+	then
+		echo failure!
+		exit 1
+	fi
+	echo \ success!
```

Comment:
what's this \ for

Reply:
It adds a space between the ellipsis after the test number and success / failure  (PASS/FAIL soon)

Reply:
I think quotes would be more obvious